enterprise/frontend/src/metabase-enterprise/moderation/actions.js

```diff
@@ -13,6 +15,7 @@ export const verifyCard = createThunkAction(
       text,
     });
 
+    dispatch(Api.util.invalidateTags([listTag("card")]));
```

Comment:
We need to change `verifyItem` and possibly `removeReview` to use RTK query mutations (via redux actions, like we do in entities). And these mutations should invalidate `card`s, as usual.